README.md

```diff
@@ -107,69 +107,67 @@ bin/rails generate vcf:builder AnotherCustomFormBuilder --namespace Forms::Compo
 # app/forms/another_custom_form_builder.rb
 class AnotherCustomFormBuilder < ViewComponent::Form::Builder
   # Set the namespace you want to use for your own components
-  namespace Forms::Components
+  namespace AnotherCustom::Form
 end
 ```
 
-:warning: **Everything below this line describes the future usage and is subject to change. It does not work yet as the gem is still under heavy development.**
-
-Now let's generate your own components to customize the rendering.
+Now let's generate your own components to customize their rendering. We can use the standard view_component generator:
 
 ```console
-bin/rails generate vcf:component Form::TextField
+bin/rails generate component Custom::Form::TextField --inline --parent ViewComponent::Form::TextFieldComponent
 
       invoke  test_unit
-      create  test/components/form/text_field_component_test.rb
-      create  app/components/form/text_field_component.rb
-      create  app/components/form/text_field_component.html.erb
+      create  test/components/custom/form/text_field_component_test.rb
+      create  app/components/custom/form/text_field_component.rb
 ```
 
+:warning: The `--parent` option is available since ViewComponent [`v2.41.0`](https://viewcomponent.org/CHANGELOG.html#2410). If you're using a previous version, you can always edit the generated `Custom::Form::CustomTextFieldComponent` class to make it inherit from `ViewComponent::Form::TextFieldComponent`.
+
 Change your forms to use your new builder:
 
 ```diff
 - <%= form_for @user, builder: ViewComponent::Form::Builder do |f| %>
 + <%= form_for @user, builder: CustomFormBuilder do |f| %>
 ```
 
-You can then customize the behavior of your `Form::TextFieldComponent`:
+You can then customize the behavior of your `Custom::Form::CustomTextFieldComponent`:
 
 ```rb
-# app/components/form/text_field_component.rb
+# app/components/custom/form/text_field_component.rb
+
+class Admin::Form::TextFieldComponent < ViewComponent::Form::TextFieldComponent
+  self.tag_klass = ActionView::Helpers::Tags::TextField
```

Comment:
@Spone I think this line may be removed

Reply:
If I remove it I get:

```
RuntimeError in Admin::Pages#edit
`self.tag_klass' should be defined in Admin::Form::TextFieldComponent
```

Reply:
My bad, I wasn't on master.

Reply:
Weird. https://github.com/pantographe/view_component-form/pull/87 was to fix that :thinking: